vignettes/articles/creating-ards-for-stats-methods.Rmd

```diff
@@ -0,0 +1,149 @@
+---
+title: "Creating ARDs with statistical methods"
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+The basic functions offered by {cards} can create a wide variety of ARDs. However, sometimes we may need to include the outputs from more complicated statistical methods in our ARDs. In this article we'll look at a few different ways to implement the output from these statistical methods.
```

Comment:
Can we put each sentence on a new line? It makes it easier to track changes to the article in the future.